src/Util/Json.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * This file is part of the prooph/pdo-event-store.
+ * (c) 2016-2018 prooph software GmbH <contact@prooph.de>
+ * (c) 2016-2018 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStore\Pdo\Util;
+
+use Prooph\EventStore\Pdo\Exception\JsonException;
+
+class Json
+{
+    /**
+     * @param mixed $value
+     *
+     * @return string
+     *
+     * @throws JsonException
+     */
+    public static function encode($value): string
+    {
+        $flags = \JSON_UNESCAPED_UNICODE | \JSON_UNESCAPED_SLASHES | \JSON_PRESERVE_ZERO_FRACTION;
+
+        $string = \json_encode($value, $flags);
+
+        if ($error = \json_last_error()) {
```

Comment:
This is working only because value of `JSON_ERROR_NONE` is set to 0, which evalutates to `false`.

I'd vote for explicitness with `JSON_ERROR_NONE !== $error = \json_last_error()`.